doc/src/site/sphinx/deployment/rsparkling_on_windows.rst

```diff
@@ -26,7 +26,6 @@ The script will check that you can:
 .. code:: r
 
     library(sparklyr)
-    options(rsparkling.sparklingwater.version = "SUBST_SW_VERSION") # Using Sparkling Water SUBST_SW_VERSION
```

Comment:
not required anymore, we package sparkling water inside rsparkling, no need to specify version